src/main/resources/cenuj-firebase.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Wolałbym nie mieć tego w repo

Reply:
To wiadomo, że jest do usunięcia. Ja już serwerek zestawiłem i czekam aż mi do DMZ to podłączą..